game-backend/src/helper/fields.ts

```diff
@@ -0,0 +1,42 @@
+import { State } from '../../../darwin-types/State';
+import Position from '../../../darwin-types/Position';
+import { ARENA_WIDTH, ARENA_HEIGHT } from '../../../darwin-types/Arena';
+
+export const getFreeFields = (state: State): Position[] => {
+  /* eslint no-shadow: ["error", { "allow": ["_"] }]*/
```

Comment:
Why not add this to the global eslint config? Otherwise everyone has to configure this by himself.